sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/converters/IndexActionConverter.java

```diff
@@ -56,14 +57,13 @@
             indexAction.setActionType(actionType);
         }
 
-        ObjectMapper mapper = new JacksonAdapter().serializer();
-        SerializationUtil.configureMapper(mapper);
-
         Map<String, Object> additionalProperties;
         TypeReference<Map<String, Object>> typeRef = new TypeReference<Map<String, Object>>() {};
+
+        ObjectMapper mapper = new JacksonAdapter().serializer();
```

Comment:
Given creating an `ObjectMapper` can be a heavy operation, thoughts on this class caching two mappers? One using the default configuration and the other setting inclusion to `Include.ALWAYS`.

Reply:
Will have two ObjectMapper as suggested.